pydarn/io/superdarn.py

```diff
@@ -689,6 +723,17 @@ def superDARN_file_structure_to_bytes(self, file_struct_list: List[dict]):
         allowed in the SuperDARN file type
         SuperDARNDataFormatTypeError - Wrong format data type error for the
         SuperDARN file type fields in the DMAP record
+
+        See Also
+        --------
+        missing_field_check(format_fields, record, self.rec_num) - checks
```

Comment:
Within the same module?

Reply:
Done, and added class name that referred to and cleaned up the presentation of the documentation.